spec/mailers/user_mailer_spec.rb

```diff
@@ -432,4 +432,76 @@
       end
     end
   end
+
+  describe "invited_to_collection_notification" do
+    let(:collection) { create(:collection) }
+    let(:otheruser) { create(:user) }
```

Comment:
I think I'm missing something here... do we have two users defined for this test? If not, I think we can just call this `user`.

If we do have two users, can you use an underscore here to separate words in the variable name? (It probably wouldn't hurt for us to make our coding standards wiki page a little clear about that, tbh.)

Reply:
The first user is defined by default as the author of the work and owner of the collection, this is the second one who is being invited by the first one.